recipe/meta.yaml

```diff
@@ -1,26 +1,30 @@
-{% set version = "1.3.1" %}
+{% set name ="pytest-console-scripts" %}
+{% set version = "1.4.0" %}
 
 package:
   name: pytest-console-scripts
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/p/pytest-console-scripts/pytest-console-scripts-{{ version }}.tar.gz
-  sha256: 5c6c3daae9cf9fbed0e655072128938600193dc002a5cf1b187248644ba02857
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6335679ef013fa20402dfd5b07219877d874bd3f0a198b84b23f17897ccbe50a
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: true # [py<37]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
     - pip
-    - python >=3.6
-    - setuptools-scm
+    - python
+    - setuptools
+    - setuptools-scm 7.1.0
```

Comment:
To clarify my understanding and reiterate the discussion in our meeting this morning, `conda-build` will automatically pull the latest version we have of `setuptools-scm`, which in this case is `7.1.0`, so we are pinning it to that version for transparency?

Reply:
I might have missed that discussion. The linter complains about it not being pinned. Is that a false positive?

Reply:
It probably should be an exception, but pinning it won't hurt.